### PR TITLE
Add xrdcp to installation bin directory

### LINKED ISSUE #7441: xrdcp CLI tools should be installed or disabled
Even though it's build and can be found in the build directory, it's not installed anymore. xrdcp is the default way to download files through the CLI, so I'd vote for having it back as part of the ROOT installation.

### PR DESCRIPTION
**Pull request by vepadulano**
Fixes #7441

### CODE REVIEW

**Review by oshadura — APPROVED** [review]

File: cmake/modules/SearchInstalledSoftware.cmake [hunk 1]
```diff
@@ -964,6 +964,15 @@ if(builtin_xrootd)
   set(XROOTD_CFLAGS "-DROOTXRDVERS=${XROOTD_VERSIONNUM}")
   install(DIRECTORY ${XROOTD_ROOTDIR}/${XROOTD_LIBDIR}/ DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT libraries FILES_MATCHING PATTERN "libXrd*")
   install(DIRECTORY ${XROOTD_ROOTDIR}/include/xrootd/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} COMPONENT headers)
+  if(NOT WINDOWS)
```
Comment (eguiraud) [inline cmake/modules/SearchInstalledSoftware.cmake:967]: @bellenot do we need this here? I guess `builtin_xrootd` is always going to be OFF on windows?)

Comment (bellenot) [reply cmake/modules/SearchInstalledSoftware.cmake:967]: No, I think it's useless. And `builtin_xrootd` is OFF by default (even if one can try to set it to ON)

**Comment by stwunsch** [comment]
Let's move the discussion whether we should install or disable the xrootd CLI tools to #7441.

**Comment by vepadulano** [comment]
Closing this after team discussion https://github.com/root-project/root/issues/7441#issuecomment-821060362

**Pull request closed without merging**